### PR TITLE
Add information about Glif Nodes subgraphs

### PR DESCRIPTION
**Pull request by ArseniiPetrovich**

### CODE REVIEW

**Comment by ArseniiPetrovich** [comment]
@trruckerfling please verify

File: smart-contracts/advanced/fevm-indexers.md
Comment (BigLep) [inline smart-contracts/advanced/fevm-indexers.md:40]: I'll let others decide as I don't know the convention here, but this largely feels like docs that Glif should own and should be on their site.  I would imagine filecoin docs just surfaces that this is available but then links off to their docs with the tutorial/example steps.

Basically, I assume we aren't going to do this for each subgraph provider.  We'd instead just have a list of the providers and where you can go to learn more.

File: smart-contracts/advanced/fevm-indexers.md
Comment (BigLep) [inline smart-contracts/advanced/fevm-indexers.md:91]: If you get rid of the extra heading above like I proposed, this would need to be changed.

**Review by BigLep — COMMENTED** [review]
These are more just drive-by comments as someone interested to learn more and as the creator https://github.com/filecoin-project/filecoin-docs/issues/2326 .  This isn't an area of expertise for me so will leave it to others to give the approval.  I do think this meets the done criteria of https://github.com/filecoin-project/filecoin-docs/issues/2326

File: smart-contracts/advanced/fevm-indexers.md
Comment (ArseniiPetrovich) [reply smart-contracts/advanced/fevm-indexers.md:45]: Ah yeah, a leftover after I changed the doc structure a bit. Thanks!

File: smart-contracts/advanced/fevm-indexers.md
Comment (ArseniiPetrovich) [reply smart-contracts/advanced/fevm-indexers.md:40]: Well, I understand your concern, it looks pretty valid. 
We are currently working on our version of the docs and it might make sense to migrate off eventually, although since there are not that many other teams competing over providing this service maybe it might make sense to keep it here for now, which would allow for two things:
1) Onboard more developers who might want to jump into Filecoin subgraphs before devcon
2) Grant some time for us to develop our own docs and eventually migrate it there, because right now there is no place on our end to store this, and despite the fact we have a solid UI, the UX kinda... in development.
But in like month or so - I'm totally up to migrating this. Or if there are new public subgraph providers emerge and want to have a bit more equal competition, like we have for RPCs.

Anyways, will let the repo maintainers decide.

**Comment by ArseniiPetrovich** [comment]
@BigLep  Thanks, I appreciate all your comments and suggestion, Steve, they are super valuable!

File: smart-contracts/advanced/fevm-indexers.md
Comment (BigLep) [reply smart-contracts/advanced/fevm-indexers.md:40]: Makes sense.  Inlining this for now seems great to me until there is a better home or other competing providers.  Good stuff - way to be pragmatic!

File: smart-contracts/advanced/fevm-indexers.md
Comment (xBalbinus) [inline smart-contracts/advanced/fevm-indexers.md:32]: LGTM. @trruckerfling need some help with merging.

Non-blocking nit: The documentation itself is limited in scope, so I would actually consider simplifying these sections even further (by removing the sentence `Although it is not...`)

**Review by xBalbinus — APPROVED** [review]

**Comment by longfeiWan9** [comment]
@ArseniiPetrovich we need to change a few files to show the Subgragh on the side navigation bar.  Can you please refer to this https://github.com/filecoin-project/filecoin-docs/pull/2295 to see how to add the link?

**Comment by longfeiWan9** [comment]
@ArseniiPetrovich  I have made some minor changes to your PR. As long as you add links to the side navigation bar, I think we are good to go.

**Comment by ArseniiPetrovich** [comment]
@longfeiWan9 @xBalbinus 
I think we should be good now

**Review by longfeiWan9 — APPROVED** [review]

**Pull request merged**
